code-challenges.js

```diff
@@ -8,62 +8,162 @@
+// use MLB.length in console.log 
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
-// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
+console.log(padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns));  
+// the output is 2,5,2,2,4,6,3,5,3 however the above is only a temporary change.  To make this action a permenant one a variable needs to be created. create a variable and name it MLB and store the concated arrays in the variable, then console.log
 
+var MLB = padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)
+
+console.log(MLB);
+
+//output is ,5,2,2,4,6,3,5,3
+
+console.log(MLB.length);
+
+// output is 9
+
+
+// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
+// const currentCohort = "Charlie 2023
 // Pseudo code:
+// console.log(currentCohort.reverse());--> wasnt that simple, should have known better
+
+// Thinking for some reason about creating a function, however I feel that is not needed and just a bunch of extra steps.  I thibk it could be done, but need to keep the code simple. 
+// .reverse is the way to go for sure. 
+// my variable contains the string, I can call my variable in console.log then split, reverse, join if I want this action to be temporary.  I will make it permenant by creating a variable called backwards and storing currentCohort split, reverse, join, then console.log
 
 const currentCohort = "Charlie 2023"
+
+const backwards = currentCohort.split("").reverse().join("")
+
+console.log(backwards)
 // Expected output: "3202 eilrahC"
+// // output is 3202 eilrahC
+
+
 
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
 // Pseudo code:
-
+// I know I need to find the index of the last 42,
+// i had to google research this.  the url is https://www.google.com/search?q=what+built+in+method+in+javascript+will+give+you+the+last+index+of+an+array&oq=what+built+in+method+in+javascript+will+give+you+the+last+index+of+an+array&aqs=chrome..69i57.16836j0j15&sourceid=chrome&ie=UTF-8  
+// I have an array, and a variable.  I will create a variable to be called on that will store the last indexed array and the givenValue1
+// the answer variable will call the array with the lastIndexOf built in method and pass in the arguement of givenValue1
+//                            0   1   2  3  4  5   6   7   8
 const numberOfConnections = [13, 34, 42, 5, 5, 10, 27, 42, 10]
 
 const givenValue1 = 42
-// Expected output: 7
+
+const answer1 = numberOfConnections.lastIndexOf(givenValue1)
+console.log(answer1);
+// output is 7
+// // Expected output: 7
+
+// repeated the same as above
 
 const givenValue2 = 10
+const answer2 = numberOfConnections.lastIndexOf(givenValue2)
+console.log(answer2);
```

Comment:
Things to keep in mind:
Whenever you store logic inside a variable just to return it (or console.log it), you can always just skip creating the variable entirely and just go straight to the return or console.log (i.e `console.log(numberOfConnections.lastIndexOf(givenValue2)`)

When you write pseudo code, try to make it as clear and concise as you can. You can remove the extra fluff that isn't directly explaining how your code is working!